### DISCUSSION TITLE
The Racism of the Remnant, and Other Stories

### DISCUSSION THREAD

**Discussion by leklachu** [endless-sky/endless-sky#7221] [General]
...there's a nice, controversial title to get @Zitchas' attention ;-)

To start off, I want to say I love the Remnant, I love their story, I love their faction. And they're someone else's creative work, not my own - so whatever way they go I'll enjoy them for what they are. The thoughts below are just my opinions and ideas.

I also want to say, if it's going in the direction of pseudo-quantum-mechanics many-worlds, I sufficiently dislike the multiple/infinite parallel universes concept to hope it doesn't become ES canon. (Again, just my opinion.)

So, playing through I have a bunch of thoughts, most which aren't big enough or concrete enough to raise a full 'issue' for, so I thought I'd compile them here, and @Zitchas (or anyone) can browse through them and consider them if you like. I'm happy to make PRs for any where helpful.

---

**The Racism of the Remnant.**

The don't-scan-me is a neat concept, but has some quirks. I built a habit of scanning as I wander round the galaxy, so quite a shock (even though I knew it was coming!) when I scan the Remnant at first sight, and suddenly they turn on me. One might consider it's an act of war, that invites me to shoot them back. Mainspace human culture has no concept of scanning being a hostile action unless you're hiding something illegal; only a little rude perhaps; but shooting is hostile and with probable intent to kill.

The cross-culture part of me thinks this is pretty cool: plenty of cultures have things you do or don't do, that seem indefensibly wrong to another culture.

But then I land, and some dude comes up and gets TikTok video all round my ship. Dude! What the ****?! Then I notice every ship around scans my ship too. In character, both of these are highly offensive (see below). Out of character, this second one is incongruous: if scanning without privilege is an act of hostility/war, then to scan a foreign ship is to treat their entrance into Remnant space as a hostile threat. At best it's naïve.

Which leads to my consideration: that again makes my cross-culture brain go, "hey, this is realistic...ish, and I can have patience with these guys." If you don't scan, the Remnant treat you as a friend, and gladly even accept your help ...while mistrusting you. This is classic racism/xenophobia: to treat a sojourner in your community in a way that you would consider it unacceptable for them to treat you: thinking on the one hand, "oh it's great to have them here and we're all lovely and friendly," and on the other, "we have to be careful of this outsider," and acting like everyone should be okay with that.

There's much more nuance (otherwise why would there be debates about what racism really is), so I'll blame any deficiency in my writing on lack of space (obviously my opinions are perfect and infinite). But that's the feeling I get in Remnant space, and out of character I make my character push through with it patiently, because I like the Remnant and want to play out their story and (in-character) be part of their community, albeit a transient part.

I'm afraid all my ideas for 'fixing' it are less-than-half-baked, so I'll leave it at that observation for now.

**Missions thick and fast**

I feel like missions come thick and fast in Remnant space, once the story starts going. For ES, I prefer the way e.g. the Wanderers paces things out a bit. I realise there's a balance to not make all these different, interesting things, take forever to happen. Some of it seems to be an artefact of developing and re-developing the story as it goes. Maybe once a major section is finished and fleshed out, including the transition to the next section, some thought could go to re-padding/pacing the first section.

The funniest was when I took Taely for a hoverboarding trip, but before we left Remnant space  I landed on a different planet to ask about something in the Graveyard.... luckily Taely "wasn't hard to find in the spaceport." I hope not. She's supposed to be on my ship!

It also felt a bit confusing about when I was really integrated with and trusted by society: being given a bunch of missions before I'd become much integrated at all.

**The Excellence of the Remnant**

I feel like the Remnant suffer from trying to have the best of everything, a bit like when I make an overpowered cheat ship then try to tone it down: but still go "but I need this, and I should have this, and this has to be this good..." Case in point, the Peregrine, felt like the game-approved ship of exactly what I'd been trying to cheat myself. (Sorry, I still prefer my unrealistically-modded Gull, once I'd perfected the design.) Maybe that shouldn't be a complaint; maybe that's just genius, and a fantastic thing to find as a player.... or maybe it feels like to have the perfect ship you have to go Remnant.

Bigger for me, is the gaslining. The cloak at least you have a cloak (albeit inferior) that you can put on any one ship. Gaslining there's no choice: you fly Remnant or miss out. That made sense for the little Puffin, doing those special missions, and was extremely cool to then get a bigger, more versatile ship with it (in the form of the Merganser): leading to the wish to have gaslining on a ship-of-my-choice. If the Peregrine is your perfect ship, then you're sorted. If not... then better hope you don't get FOMO (Fear of Missing Out). I mod my ship to have gaslining, because I get FOMO.

After the Peregrine is completed, and as you expand landable gas giants (which I hope you will!), could we get a gaslining kit as an outfit, that could set up a pressure-capable living section on any [flag]ship?

**One-liners**

- I got the choice to reveal or not reveal the Remnant to the Deep, before I really had any reason not to. But I was afraid of the consequences so forced my character to choose not-reveal
- The Crippled raiders keep jumping away. Well, maybe I just have to invest in a ship/fleet with more 'stopping power'... but I'd have preferred to be allowed to whittle it down without a wild goose chase
- I plundered and landed with several Broken Jump Drives before I got the mission. Maybe it's being crowded out by other `priority` missions?
- The Archon in Nenia, early on, "attacks Remnant ships". But really attacks by mission. I probably can't legitimately have a Remnant ship before the mission (unless I stole one? Maybe the Archon's like, "heh, you killed the Remnant, I'm cool.") but I can definitely fly to Nenia in something other than the Puffin while it's hostile. Dunno how to game-mechanics-fix this one.
- mission: "Remnant: Technology Available", `remnant 1 introduction`, L797. You've made it invisible, but have a name and a description. Did you mean that? So I got told, "let's head over to <destination>", but then had no mission in my log to remind me which planet to go to. 
- `remnant ` introduction` L3785, the ion storm. "Half a millennia[sic]." -> Half a millennium!
- `remnant 2 cognizance`, L687 and your comment. I believe your problem is `neighbor distance 1 5` means: the stopover should be planet which is a neighbour of a system 1-5 jumps away. You can ditch the `not distance 0` and `neighbor` and just have `distance 1 5`
-  `cognizance` mission Remnant: Cognizance 20, I think there's a (non-critical) loophole: if you pilot a puffin, but your escorts have space for 10 cargo, you can get stopover on Esquiline. Then you've got to either swap flagship *after* the stopover to fit the research lab, or just shove the lab in your cargo (that's a wider issue with outfit missions, I plan to bring up in a separate issue when I get round to it)
- `kahet missions` L258, bringing the Quarg bit to the Quarg. No matter how many times I try, I can't make sense of the grammar: "[they] attempted to relocate... in worlds easier to maintain than ones they were closer to"
- due to mechanics of deprecation, the brand new Peregrine you get, is only worth 25% if you want to sell it or some outfits. I don't think there's a workaround though, without changing game mechanics. Must have been a display model: I guess I should have known the Remnant wouldn't be generous enough to give me a fresh, mint-condition ship! ;-p
- more Peregine: "There's a woosh and the Peregrine reappears." That's some big whoosh, from a 900-ton warship
- The monthly remembrance ceremony, that Taely's going to this month... happens several times a month and people usually take part in one every few weeks.

**Ssil Vida**

How do they communicate with the rest of Remnant space? Clearly not by human-style hyperlink network. This confused me when Ssil Veda asked to take a data crystal but said they'd already sent the data. Could it be worth playing with the idea that comms are only through the hyperlink network, so Ssil Vida only has communication with main Remnant space through keystone/jumpdrive-equipped ships bearing messages? Could be fun?

Also after switching the station on and off again ("please restart Ssil Vida for the settings to have effect"?) the jump distance from those systems is still halved. Do you intend that?

**Remnant and the wider galaxy**

How much do the Remnant really interact with the wider galaxy? At first it seems they're tucked in their corner of space, and you're their first contact from back home. The data archive they ask for from the Deep IIRC is that they remembered some effort was starting to make that, hundreds of years ago. Then it turns out they're scouting the whole galaxy, and have access to the communication network to place orders and stuff. It feels like their culture is first defined round them being cut off and in the dark about 'Ancestral Space', then grafts on and retcons all this wider activity and knowledge.

And do they place orders direct from their computers at home? Or have to have an agent in human space to connect into the network to place the orders? Can they communicate directly with people in human space? As a note: for (I presume) game-engine-reasons, it's mostly treated that hyperspace communications happens only on planets, and in-system comms can happen to off-world ships. Have the Remnant expanded this to inter-system ship-based comms?

When you take Taely joyriding, and before that the three scientists to Tarazed, is this a big new first for Remnant community? Or do some of them head off to visit human space, and human land, from time to time anyway?

They also know I discovered the Emerald Sword. Did I tell them? I guess I must have. Or... they're trying to rival the Deep and Republic Intelligence with their spy network. Do we get a story arc where the Deep and Remnant are each trying to infiltrate each others' networks, and which one you help ultimately grows into the fourth-millennium Illuminati?

**Remnant Orthography**

"Now that you've gotten used to us, we'll show you our real outfit names." Personally I think this could be quietly ignored, but if not, raises a more intriguing - and I think very cool - question. Remnant orthography (writing). Amongst deaf communities, so I have heard, written English is hard for many because the phonemes made by the letter groups don't mean anything, so written words are rather like arbitrary telephone numbers. Some written/screen projects for the deaf have tried essentially a printed version of sign language I think, or an animated avatar that signs the text. (I'm afraid I don't know what the end result was, or if this technique is used widely.)

Now, most Remnant are still capable of hearing, and still remember American language from the good ol' American galaxy. I wonder if their sign language is a more-or-less direct map to spoken language, like I heard of  a Spanish community that invented a whistling to pass messages unbeknown to the their oppressors - but maps directly to normal Spanish if you know. Or like real sign languages which, I believe, are more distinct languages of their own (though I presume still closer in thought-patterns than a language from a distant language family). 

Which leads to the question: what is the Remnant orthography? When you read the 'proper names', are you reading them in Roman (or 'galactic common') script? Are they transliterated from Remnant (...which doesn't have phonemes or sound-based letters, so that doesn't seem to work); are they translated (then what does being the 'real Remnant name' mean); or is there a standard way of mapping Remnant signs to sounds, similar to how Chinese characters mean a conceptual word more than they mean a spoken word but can still be used as their associated sounds to transliterate either way.

Or, has the player learnt a whole new orthography, the one the Remnant use to write down their language (if they don't write in 'ancestor' script)?

**The Dromedary**

I'm so looking forward to playing through this new mission. Just found the ghost ship... it's so cool.
- but its engines fire fully. You could give it intrinsic movement (like the merganser, but large) and either no engines or small ones to look more ghostly
- I want to fly a ghost ship! It shall be the Flying Dutchman, and I shall ferry the souls of discarded NPCs through the Pug wormhole to their final rest.
  - (Side note: please, no one actually write an afterlife or body/soul separation into ES canon. It's another concept I find clunky when dealt with in fiction and feel it's out of place for ES.)

**A bug**

I'm sitting on a bug of, what looks to be, an event-caused change in the savefile, "Did not find matching variant for specified operation: fleet "Small Remnant" ... remove variant 1" and same for Large Remnant variant 4.

I'll... try and track it down when I have energy to, and file a proper bug report or PR. It could be from updating parts of the base map and the matching event-to-change, making a mismatch with savegames that have an earlier version of the event. If so, I wonder if it's worth developing more in plugin, then committing to main only what you're sure not to change (except bugfixes obviously).

**Finally**, I want to say thank you again for creating this, it's a lot of fun and a great part of the ES universe.

**Comment by Zitchas** [comment]
The Scanning:
- It sounds like you haven't reached the "permission to scan" mission yet. At a certain point, probably fairly early on for most people, they'll get permission and even encouragement to scan Remnant ships. There is also a lot more dialog options/mentions in their hails about *not* scanning them up until that point. 

Gaslining/Cloaking
- I actually dislike the cloaking outfit as it cheapens cloaking, and forces content creators to assume that no matter what the player is flying, it has a significantly likelyhood of cloaking. So, does the mission make sense if the player can cloak? Does it make sense when the player can sneak their super-heavy warship into wherever they need to go?
- The Peregrine's gaslining ability is there because the Remnant do ocassionally need to visit such worlds with a warship. But it is *also* there so that players who want to explore the galaxy and see all the other gaslined requiring worlds out there aren't forced into a "curiosity or something that can actually take a hit and go more than a handful of jumps at a time" dilema. (and no, there aren't any other worlds outside of Remnant territory... yet. There will be) The puffin is great for its mission, but its a real pain to use for... well... anything, really. Poor range, poor cargo, poor passengers, abysmal combat... More to the point, I didn't want to force players like you who *want* to be able to explore those worlds to save-edit or use a plug-in to get a gaslining kit in order to do so. So, the Peregrine is the ship for you, with great agility, good range, good combat ability, and sufficient cargo & passengers to at least do some mission running.  It's not a great freighter or transport by any means, but it's something, anyway.

Unfortunatley, I don't have the time to go through the rest point by point right now, but I appreciated reading them.

**Comment by leklachu** [comment]
I've had the permission to scan mission. And seen the 'please don't scan me' hail - but that only because I knew to look for it. I don't have a habit of hailing ships, and it took me several tries to get that message.

The permission to scan dialogue made it make a bit more sense, but still there's this willingness to do, to the player, an action that would be hostile if the player did it to them. "Only we can do this until we trust you." Uh-huh. Remind me why I fought off Korath raiders with you, and helped you with emergencies, then?

If you play stronger into a real fear mindset of "everyone might be the enemy until we're really sure", like the blood test does, that would make more sense to me: but for that I think it'd need to be fleshed out more through the culture and interaction, and maybe more care taken about how you ever manage to end up on their good side in the first place.

---

Gaslining/cloaking: The thing is, should these things be exclusive to the Remnant? I can bring a Coalition ship to help the Wanderers in Korath space: I'm allowed that creative freedom. What if I love the Strong Wind. Can I play through the Remnant with that? Can I explore the outside-Remnant-space gas giants without buying/inheriting a Remnant-exclusive ship? I see pros and cons both ways, but for the fun of exploration and creative freedom, I favour being able to do these things in a different ship, except for specific missions.

Since not all gas giants are landable, but some are before you have an explicit mission for them: Could there be a way of becoming aware of them while you fly in another ship, so you could go back in a gaslined ship to look later? Part of my problem is I don't want to do a dedicated sweep through the galaxy for things I've missed that I can only find in one ship. (e.g. I keep a keystone on me always once I know about them.)

As to the cloak specifically, it's such an enormously powerful ability... well, I think so anyway. I liked when just the gull and starling had cloak, because, yes you can now get a fleet of infinite-cloakers (with a bit of ramscoop), but on the compromise of smallish ships. Again, I have feelings both ways, and having the Peregrine with inbuilt cloak is awesome... but then I ask, why should I ever fly something non-Remnant? Maybe if each other species (including regular humans!) had something very special and unique that you couldn't fit into another ship, it'd feel more balanced. (Like the Dragonflame: though that has its own downsides).

I see your point with cloaking device cheapening cloaking. I feel the same about Jump Drives...
At the same time, being able to buy a fleet of cloaked ships is what first cheapened cloaking to me, compared to one and only one ship can have a cloak.
